src/system/function.ts

```diff
@@ -98,13 +97,6 @@ export function debounce<T extends (...args: any[]) => ReturnType<T>>(
 		lastThis = this;
 		lastCallTime = time;
 
-		if (isInvoking) {
-			if (timer == null) {
-				// Start the timer for the trailing edge.
-				timer = setTimeout(timerExpired, wait);
-				return result;
-			}
-		}
```

Comment:
Can you elaborate why this logic is being removed?

Reply:
@axosoft-ramint 

This piece of code is removed because it did nothing. It didn't break thing, though. Looks like there used to be a special edging case handling that have been removed later. And this code looks like a piece of logic that just hasn't been removed but without special execution for edging case it does nothing.